spec/index.bs

```diff
@@ -358,8 +358,25 @@ Shader Module {#shader}
 // text and binary input.
 typedef (Uint32Array or DOMString) GPUShaderCode;
 
+dictionary GPUShaderSemantic {
+    required DOMString type; // "u", "s", "t", "b"
+    required u32 binding;
+    u32 space = 0;
+};
+
+dictionary GPUBindPoint {
+    required u32 binding;
+    required u32 bindGroup;
+};
+
+dictionary GPUOverriddenBindPoint {
+    required GPUShaderSemantic shaderSemantic;
+    required GPUBindPoint overriddenBindPoint;
+};
+
 dictionary GPUShaderModuleDescriptor : GPUObjectDescriptorBase {
     required GPUShaderCode code;
+    sequence<GPUOverriddenBindPoint>? overriddenBindPoints = null;
```

Comment:
nit: this member is already optional because it isn't marked `required` so we shouldn't need the question mark.

Reply:
oops, same-ish as my comment below. marking resolved